src/main/java/nl/thehyve/ocdu/validators/patientDataChecks/SitePatientDataCheck.java

```diff
@@ -0,0 +1,28 @@
+package nl.thehyve.ocdu.validators.patientDataChecks;
+
+import nl.thehyve.ocdu.models.OCEntities.Subject;
+import nl.thehyve.ocdu.models.OcDefinitions.MetaData;
+import nl.thehyve.ocdu.models.errors.ValidationErrorMessage;
+import org.apache.commons.lang3.StringUtils;
+
+/**
+ * Created by bo on 6/15/16.
+ */
+public class SitePatientDataCheck implements PatientDataCheck{
+
+    @Override
+    public ValidationErrorMessage getCorrespondingError(int index, Subject subject, MetaData metaData) {
+
+        String ssid = subject.getSsid();
+        String commonMessage = getCommonErrorMessage(index, ssid);
+
+        ValidationErrorMessage error = null;
+        String site = subject.getSite();
+
+        if(!StringUtils.isBlank(site)) {
+            //TODO: check if site exists in study
+        }
+
+        return error;
```

Comment:
Convention we followed so far was that Checks object returns error when there is error in the data and null when it is fine. In this case you will always return error. But I guess it is because it is only a draft, but just checking.